legend-engine-xt-relationalStore-sqlserver-execution-tests/src/test/resources/org/finos/legend/engine/server/test/userTestConfig_withSqlServerTestConnection.json

```diff
@@ -16,6 +16,7 @@
       }
     ]
   },
+  "sessionCookie": "LEGEND_ENGINE_JSESSIONID",
```

Comment:
Please remove all changes from userTestConfig_with*TestConnection.json files. They are not used with Server, but with RelationalTestServer, which is minimal server used for testing.